### ISSUE TITLE
Feature Request: Improve usability of CreateLookupVindex command

### ISSUE THREAD

**Issue by mattlord** [vitessio/vitess#13960]
### Feature Description

The current known "rough edges" for the command are:

1. The CreateLookupVindex workflow fails if related artifacts like vschema/schema entries already exist on the target
2. If a previous (failed) CreateLookupVindex workflow still exists then when creating a new one you get an error that a frozen workflow exists on workflow. This was added as protection to force cleanup of previous workflows to avoid restart of older workflows on error, for example. But this prevents easily running multiple workflows in parallel, so the validations need

### Use Case(s)

Managing many lookup vindexes in Vitess.

**Comment by maxenglander** [comment]
> If a previous CreateLookupVindex workflow still exists when creating a new one then you get an error that a frozen workflow exists.

I am able to issue multiple `CreateLookupVindex` one after another without issue. After creating them and running `Workflow <workflow_id> Show`, they all show as running without error. I can then "complete" them via `ExternalizeVindex` one after the other, and that produces the desired effect (delete VReplication streams, remove `write_only` param) as well.

Also, not sure what causes a `CreateLookupVindex` workflow to go into frozen state. I only know about creating them, letting them run, and then deleting them via `ExternalizeVindex`.

**Comment by mattlord** [comment]
Thanks, @maxenglander ! That list may be outdated, I'm not sure. I was starting with an aggregated list of things that I've seen reported. There may be more to at least some of them.

**Comment by lizztheblizz** [comment]
@mattlord - Perhaps I need to open a separate issue, but another limitation in `CreateLookupVindex` is that it will not accept multi-column lookup vindex definitions, even though they do exist and function perfectly well when manually created.

i.e. this is a statement that will fail with the following error:
```
CreateLookupVindex ks '{"sharded":true,"vindexes":{"id_test_idx":{"type":"consistent_lookup_unique","params": {"from": "id,test_column","table": "ks.table_name","to": "keyspace_id"},"owner": "table_name"}},"tables":{"table_name":{"column_vindexes":[{"columns":["id","test_column"],"name":"id_test_idx"}]}}}'
```

Returns error:
```
unique vindex 'from' should have only one column: type:"consistent_lookup_unique" params:{key:"from" value:"id,test_column"} params:{key:"table" value:"ks.id_test_idx"} params:{key:"to" value:"keyspace_id"} owner:"table_name"
```

While the equivalent `alter vschema` statement works perfectly fine, and produces a functioning vschema:
```
alter vschema on ks.table_name add vindex id_test_idx(id, test_column) using consistent_lookup_unique with owner=`table_name`, table=`ks.id_test_idx`, from=`id,test_column`, to=`keyspace_id`;
```
